docs/streaming-kafka-0-10-integration.md

```diff
@@ -12,6 +12,8 @@ For Scala/Java applications using SBT/Maven project definitions, link your strea
 	artifactId = spark-streaming-kafka-0-10_{{site.SCALA_BINARY_VERSION}}
 	version = {{site.SPARK_VERSION_SHORT}}
 
+**Do not** manually add dependencies on org.apache.kafka artifacts (e.g. kafka-clients).  The spark-streaming-kafka-0-10 artifact has the appropriate transitive dependencies already, and different versions may be incompatible in hard to diagnose ways.
```

Comment:
I suppose that if the app is using kafka APIs directly it should declare a direct dependency on it, but, harmonize the versions. That said, that's error prone, so this seems OK. If you make further changes to the text here can you back-tick the artifact names for readability?